DNN Platform/Library/Entities/Users/UserEventArgs.cs

```diff
@@ -26,6 +26,6 @@ namespace DotNetNuke.Entities.Users
     public class UserEventArgs : EventArgs
     {
         public UserInfo User { get; set; }
-        public bool SendNotification { get; set; }
+        public bool SendEmailNotification { get; set; }
```

Comment:
if UserEventArgs is part of the public API, this would be a breaking change

Reply:
fixed!